api/api/models/media.py

```diff
@@ -134,16 +135,13 @@ class AbstractMediaReport(models.Model):
     Generic model from which to inherit all reported media classes.
 
     'Reported' here refers to content reports such as sensitive, copyright-violating or
-    deleted content. Subclasses must populate ``media_class``, ``sensitive_class`` and
-    ``deleted_class`` fields.
+    deleted content. Subclasses must populate fields ``media_class`` and ``url_frag``.
     """
 
     media_class: type[models.Model] = None
     """the model class associated with this media type e.g. ``Image`` or ``Audio``"""
-    sensitive_class: type[models.Model] = None
-    """the class storing sensitive media e.g. ``SensitiveImage`` or ``SensitiveAudio``"""
-    deleted_class: type[models.Model] = None
-    """the class storing deleted media e.g. ``DeletedImage`` or ``DeletedAudio``"""
+    url_frag: str = None
```

Comment:
Can we use the full name, here, rather than an abbreviation? Or else `url_path_part` might be more descriptive still.

I'm fairly certain it's unnecessary if we use `reverse` anyway, though. Even if we still need some reference to the plural media type, ideally this kind of metadata is defined on the `media_class` or in something like the `MediaTypeConfig` dataclasses we have in the test suite, as it's information shared by all related models.